opm/autodiff/StandardWellV_impl.hpp

```diff
@@ -679,7 +680,8 @@ namespace Opm
                     const unsigned int compIdx = flowPhaseToEbosCompIdx(p);
                     const double drawdown = well_state.perfPress()[first_perf_ + perf] - intQuants.fluidState().pressure(FluidSystem::oilPhaseIdx).value();
                     double productivity_index = cq_s[compIdx].value() / drawdown;
-                    scaleProductivityIndex(perf, productivity_index, deferred_logger);
+                    bool new_well = schedule.hasWellEvent(name(), ScheduleEvents::NEW_WELL, current_step_);
```

Comment:
Please constify new_well here (also in StandardWellsV), and where used as function argument (apart from correctness issues, that is the style of the existing code).

Reply:
OK - done; Jenkins was green - I'will not rerun after the `const` fixup.